src/overlays/actors/ovl_Obj_Ending/z_obj_ending.h

```diff
@@ -5,9 +5,16 @@
 
 struct ObjEnding;
 
+typedef struct {
+    /* 0x0 */   UNK_TYPE4 unk0;
+    /* 0x4 */   Gfx* dl;
+    /* 0x8 */   UNK_PTR unk8;
```

Comment:
Out of curiosity, how do you know this is an AnimatedTexture pointer? Is it because it being passed into `Lib_SegmentedToVirtual` which result is then a pointer to an AnimatedTexture?

Reply:
Yes, that's exactly how I knew.